config/config.go

```diff
@@ -34,19 +35,19 @@ func LoadConfig(confName string) (*MergedConfig, error) {
 	return nil, err
```

Comment:
Somehow this `for` loop appears wrong as well, if an `err` is assigned something by config.ReadDefault it is ignored the variable `err` is created inside the for loop so it is impossible to reach line 32. I know you didnt change this part but we may as well correct its behavior.

    for _, confPath := range confPaths {
        if conf,err := config.ReadDefault(path.Join(confPath, confName)); err!=nil && err!="not found" {
              return err
        } else if err==nil {
               return &Context{config:conf}, nil
        }
    }
    return nil, fmt.Errorf("Config %s not found",confName)